src/pages/reciters/[reciterId]/[chapterId].tsx

```diff
@@ -151,7 +152,12 @@ export default RecitationPage;
 export const getStaticProps: GetStaticProps = async ({ params, locale }) => {
   try {
     const reciterId = params.reciterId as string;
-    const chapterId = params.chapterId as string;
+    let chapterId = params.chapterId as string;
+    const sluggedChapterId = await getChapterIdBySlug(chapterId, locale);
```

Comment:
quick one, maybe we only need to call `getChapterIdBySlug` only if the chapterId is not number?